src/get-started/first-java-service.md

```diff
@@ -0,0 +1,615 @@
+# First Exonum Java Service Tutorial
+
+<!-- cspell:ignore Lorean, Intelli, testnet -->
+
+This tutorial is aimed at beginners in Exonum Java. It is an introduction
+into service development in Java. As such, it does not cover some important
+to real-world services topics, like authorization, or proofs of authenticity
+for service data, or testing, but gives a foundation to learn about that
+in subsequent materials.
+
+<!-- todo: Is it the right introductory page on main Exonum principles and
```

Comment:
I think yes; the *What is Exonum?* and *Design Overview* articles are the two introductory pages.